composer.json

```diff
@@ -282,6 +282,9 @@
             "drupal/redirect": {
                 "Match redirect": "https://www.drupal.org/files/issues/2020-04-13/redirect-n2831605-31.patch"
             },
+            "drupal/replicate": {
+                "Replicate components in the proper order.": "https://git.drupalcode.org/project/replicate/-/merge_requests/8.diff"
```

Comment:
This should be brought locally and stored in the patches directory so that if it changes upstream we won't automatically accept the changes.